client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,223 @@
  * External dependencies
  */
 import { bindActionCreators } from 'redux';
-import { localize } from 'i18n-calypso';
+import { translate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import { fetchEmailSettings } from 'woocommerce/state/sites/settings/email/actions';
 import {
-	fetchEmailSettings,
-} from 'woocommerce/state/sites/settings/email/actions';
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import Card from 'components/card';
+import CustomerNotification from './components/customer-notification';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+
+const fromName =
+	{
+		field: 'woocommerce_email_from_address',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inpboxses \'from\' this name.' ),
```

Comment:
The subtitle here should be

`Emails will appear in recipients inboxes 'from' this name`